pkgs/top-level/all-packages.nix

```diff
@@ -6820,7 +6820,7 @@ with pkgs;
     zig = buildPackages.zig_0_12;
     inherit (darwin.apple_sdk.frameworks) CoreServices Security;
   };
-  cargo-msrv = callPackage ../development/tools/rust/cargo-msrv {
+  cargo-msrv = callPackage ../by-name/ca/cargo-msrv/package.nix {
     inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
I think this is not necessary anymore, you can just remove `Security` from the package and it should still work.

Reply:
(Also `libconv` is probably not required anymore for Darwin).

Reply:
I will try this and let you guys know if this works.